tests/test_directives.py

```diff
@@ -119,3 +121,24 @@ def test_code_description_missing_column(app, status, warning):
     assert_build(app, status, warning, basename, [
         f"Column 'non-code' not found (Code, Description): {path(basename, 'codelist.csv')}",
     ])
+
+
+@pytest.mark.sphinx(buildername='html', srcdir=path('workedexample'), freshenv=True)
+def test_workedexample(app, status, warning):
+    env = app.builder.env
+    assert (not hasattr(env, WORKED_EXAMPLES_ENV_NAME))
+    assert_build(app, status, warning, 'workedexample')
+    assert (getattr(env, WORKED_EXAMPLES_ENV_NAME)[0]['title'] == 'Unsuccessful tender')
```

Comment:
Avoid extra parentheses
```suggestion
    assert not hasattr(env, WORKED_EXAMPLES_ENV_NAME)
    assert_build(app, status, warning, 'workedexample')
    assert getattr(env, WORKED_EXAMPLES_ENV_NAME)[0]['title'] == 'Unsuccessful tender'
```